scripts/generate_project_docs.py

```
@@ -18,32 +18,31 @@
     def __init__(self):
         """Initialize the CLI."""
         self.doc_generator = get_project_documentation_generator()
         self.start_time = datetime.now()
-    
+
     async def run_documentation_generation(
-        self,
-        project_name: str = None,
-        generate_all: bool = False,
-        force: bool = False
+        self, project_name: str = None, generate_all: bool = False, force: bool = False
```

Ruff (0.13.3)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)